implementation/contracts/deposit/DepositUtils.sol

```diff
@@ -33,6 +33,7 @@ library DepositUtils {
         uint256 lotSizeSatoshis;
         uint8 currentState;
         uint256 signerFeeDivisor;
+        uint128 initialCollateralizedPercent;
```

Comment:
This and the two below seem like they can be smaller than `uint128`, perhaps? Could even be a `uint8`? Maybe a `uint16` if we want to allow ourselves maximum flexibility, but our percentages are expressed as percentages, so `uint8` gives us up to 250% as a representation 🤔 

Not sure if the Solidity compiler is good enough to pack things together and save us gas in that case though.

Reply:
`Uint16` for some peace of mind. Also, the enforced `initialCollateralizedPercent` max is 300, which is over the 255 uint8 limit. 
Solidity will automatically pack where possible, given the variables fit into a single 32-byte increment. two `uint128`s should be packed